src/index.css

```diff
@@ -252,3 +252,128 @@ body,
 .output-panel.hidden {
   display: none;
 }
+
+.log-container {
+  font-family: monospace, monospace; /* stylelint-disable-line font-family-no-duplicate-names */
```

Comment:
I believe this is equivalent to `font-family: monospace`.

Reply:
It is not. With `monospace`, both Firefox and Chrome use a smaller font size. With `monospace, monospace`, the font size is preserved. But I can set the font size explicitly.